tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/PortalBundle.java

```diff
@@ -30,15 +31,17 @@
 
     IPath getLiferayHome();
 
-    String getMainClass();
+    String getStartMainClass();
+    
+    String getStopMainClass();
 
     IPath getModulesPath();
 
-    IPath[] getRuntimeClasspath();
+    IPath[] getRuntimeClasspath(IVMInstall vmInstall);
```

Comment:
Lets remove the vminstall for all these methods.

also lets remove the getRuntimeClasspath() it is not going to be needed.